taigit/src/libraries/Taiga.ts

```diff
@@ -78,4 +78,124 @@ task_history(taskId : number) : Promise<Object> {
     }
 
     return output;
-}
\ No newline at end of file
+}
+
+/**
+ * @summary This call assess a task based on task_assessment_state_trans based on task Id
+ * @param taskId the ID for the task to assess task
+ * @returns  whether the task's state transition is valid,task status transition, date based on task Id
+ * * {
+ * Current in Use
```

Comment:
I do not believe your current in use/can be added notation are valid javadocs and am inclined to believe those are notes left to yourself?